### DISCUSSION TITLE
How to enable bash auto completion in lldpcli interactive mode?

### DISCUSSION THREAD

**Discussion by rbu9fe** [lldpd/lldpd#692] [Q&A]
I'm building myself for a Debian system and I can't get the bash completion to work in the interactive mode. When I execute `source /usr/share/bash-completion/completions/lldpcli` and then type `lldpcli TAB` then it works as expected, however, it does not in the interactive mode. Perhaps it has to do with this message I'm seeing when starting lldpcli:
```
root@ES570:~# lldpcli
Cannot read termcap database;
using dumb terminal settings.
[lldpcli] #
```
Do you have any idea how to fix that?

**Comment by vincentbernat** [comment]
This does not seem related. bash completions require the package `bash-completions` to work.

**Comment by rbu9fe** [comment]
Ok thanks, the package is installed and bash completion works in non-interactive mode, so I guess I'll have to look somewhere else.
